### PR TITLE
Bug 1656036 - Stop trying to ingest Fenix pushes for branches that we…

### PR DESCRIPTION
**Pull request by armenzg**
… don't support

There are Fenix tasks coming down the ingestion pipeline for PRs and non-master branches.

This avoids raising MissingPushException and retrying ingesting the task multiple times.

You can test this locally with this command:

```shell
./manage.py ingest task --task-id LKSAc7NLTIWQIo3g-XTybw
```

### CODE REVIEW

**Comment by armenzg** [comment]
I'm running this on prototype.

**Comment by armenzg** [comment]
With fewer workers prototype is catching up faster:
image

**Comment by armenzg** [comment]
I pushed one more time for tasks that do not use those env variables. For instance: https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/LKSAc7NLTIWQIo3g-XTybw

File: treeherder/etl/job_loader.py [hunk 1]
```diff
@@ -99,11 +102,19 @@ def validate_revision(self, repository, pulse_job):
 
         if not Push.objects.filter(**filter_kwargs).exists():
             (real_task_id, _) = task_and_retry_ids(pulse_job["taskId"])
-            raise MissingPushException(
-                "No push found in {} for revision {} for task {}".format(
-                    pulse_job["origin"]["project"], revision, real_task_id
-                )
+            repository = Repository.objects.get(name=pulse_job["origin"]["project"])
+            # /api/queue/v1/task/{task_id}/runs/{run_number}/artifacts/public/runnable-jobs.json
```
Comment (sarah-clements) [inline treeherder/etl/job_loader.py:106]: Is this comment needed?

**Pull request merged**